### PR TITLE
Add AkkaManagement Setup class

### PR DESCRIPTION
**Pull request by Arkatufus**

### CODE REVIEW

**Comment by Arkatufus** [comment]
Should we make these Setup classes immutable?

File: src/management/Akka.Management/AkkaManagementSetup.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using Akka.Actor.Setup;
+
+namespace Akka.Management
+{
+    public sealed class AkkaManagementSetup: Setup
```
Comment (Aaronontheweb) [inline src/management/Akka.Management/AkkaManagementSetup.cs:16]: Yes, we should make this immutable

File: src/management/Akka.Management/AkkaManagementSetup.cs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        /// The value will need to be from 0 to 65535.
+        /// </summary>
+        public int? Port { get; set; }
+        
+        /// <summary>
+        /// Use this setting to bind a network interface to a different hostname or ip
+        /// than the HTTP Server for Http Cluster Management.
+        /// Use "0.0.0.0" to bind to all interfaces.
+        /// </summary>
+        public string EffectiveBindHostname { get; set; }
```
Comment (Aaronontheweb) [inline src/management/Akka.Management/AkkaManagementSetup.cs:47]: Want to stick with "public port" and "public hostname" conventions that we have established in Akka.Remote?

**Review by Aaronontheweb — CHANGES_REQUESTED** [review]
Minor changes needed

File: src/management/Akka.Management/AkkaManagementSetup.cs (hunk 1 above)
Comment (Aaronontheweb) [reply src/management/Akka.Management/AkkaManagementSetup.cs:16]: Disregard - too much work for now.

**Review by Aaronontheweb — APPROVED** [review]

**Pull request merged**